### PR TITLE
Add very basic mission submission frontend

### PR DESCRIPTION
**Pull request by ppepos**
This is as basic as it gets.

TODO: ~~Update API backend to treat JSON post data instead of default raw post data.~~ (PR'd)
TODO: Integrate the runner and lang options to the ui.
TODO: Style my plain bootstrapocalypse.

But mergeable as-is...

Signed-off-by: Philippe Pepos Petitclerc ppeposp@gmail.com

### CODE REVIEW

**Comment by privat** [comment]
+1

File: www/javascripts/missions.js [hunk 1]
```diff
@@ -72,4 +93,15 @@
 				templateUrl: '/directives/player/mission.html',
 			};
 		}])
+
+		.directive('missionSubmit', [function () {
+			return {
+				transclude: true,
+				scope: {
+					missionId: '=missionId'
```
Comment (Morriar) [inline www/javascripts/missions.js:101]: Why not `=`?

**Comment by Morriar** [comment]
+1

**Comment by ppepos** [comment]
rerolled

**Comment by privat** [comment]
still +1

**Pull request merged**